agent/wsclient/client.go

```diff
@@ -132,6 +132,7 @@ type ClientServerImpl struct {
 // 'MakeRequest' can be made after calling this, but responss will not be
 // receivable until 'Serve' is also called.
 func (cs *ClientServerImpl) Connect() error {
+	seelog.Debugf("Establishing a Websocket connection to ", cs.URL)
```

Comment:
Can you use the format string here instead?  `"Establishing a Websocket connection to: %s", cs.URL`